docs/subquery_network/kepler/indexers/become-an-indexer.md

```diff
@@ -121,6 +121,13 @@ After making these adjustments, start up the docker-compose file with the follow
 docker-compose -f ./metrics/docker-compose-metrics.yml up -d
 ```
 
+Head to `http://localhost:3000`, your username will be `admin` and password will be whatever you set for `GF_SECURITY_ADMIN_PASSWORD`
```

Comment:
[question] Should be `indexer-endpoint:3000` ?

Reply:
Ah yes it should be

Reply:
```suggestion
Head to `http://<indexer-endpoint>:3000`, your username will be `admin` and password will be whatever you set for `GF_SECURITY_ADMIN_PASSWORD`
```